src/network/networkd-link.c

```diff
@@ -1567,6 +1571,13 @@ static int link_up(Link *link) {
         if (r < 0)
                 return log_link_error_errno(link, r, "Could not allocate RTM_SETLINK message: %m");
 
+        /* set it free if it's not enslaved with networkd */
+        if (!link->network->bridge) {
```

Comment:
Hmm apparently it should be `if (!link->network->bridge && !link->network->bond) {` instead?